### DISCUSSION TITLE
管理员权限启动 PCL2 无效

### DISCUSSION THREAD

**Discussion by burningtnt** [Meloong-Git/PCL#3028] [提问]
来自：#3025

**Comment by radish-andy** [comment]
好像没啥用

**Reply by burningtnt** [reply]
你确定你登录了 Administrator 账户，并将 PCL2 放入了白名单后再回到了原账户启动 PCL2？

**Reply by SqrtTan** [reply]
我也是联想，也没开Admin，用的360系列，好像没啥问题
Edit: 收回那句话，我`whoami`看了一下，我就是Admin……
![image](https://github.com/Hex-Dragon/PCL2/assets/121204926/baf15b1c-5ad9-40f4-b5a4-7e3eb196a28d)

**Reply by radish-andy** [reply]
防火墙白名单？

**Reply by burningtnt** [reply]
> 防火墙白名单？

是的

**Reply by burningtnt** [reply]
> 我也是联想，也没开Admin，用的360系列，好像没啥问题
> Edit: 收回那句话，我whoami看了一下，我就是Admin……

其实，只有 Administrator 账户有对 Windows 的完全管理权

**Comment by radish-andy** [comment]
允许应用通过-其他应用-PCL2?

**Reply by burningtnt** [reply]
请截图，并，请合理使用跟 Thread 功能，即，请在此回复
![image](https://github.com/Hex-Dragon/PCL2/assets/88144530/af987670-ab8f-47c4-96ab-2e87f56f324b)
Write a reply

**Reply by radish-andy** [reply]
![屏幕截图 2023-12-23 114339](https://github.com/Hex-Dragon/PCL2/assets/119546929/32d2e582-849b-4705-96b2-50e7d0fd7376)
更改之后已经保存了回到这边还是不行

**Reply by burningtnt** [reply]
你需要添加他到防火墙白名单，而不仅仅时网络通讯白名单

**Reply by radish-andy** [reply]
how?

**Reply by burningtnt** [reply]
https://baijiahao.baidu.com/s?id=1764022221353463916&wfr=spider&for=pc

**Reply by radish-andy** [reply]
这不是排除项吗

**Reply by radish-andy** [reply]
不太行

**Reply by radish-andy** [reply]
下午再弄，吃饭了

**Comment by WorldHim** [comment]
你的账户在 administrators 组里吗？
关掉 uac 试试

**Reply by burningtnt** [reply]
再强调一遍：他这个品牌的电脑的超级管理员账号 Administrator 是隐藏的，卖给你的时候那个账户没有完全的管理员权限

**Reply by WorldHim** [reply]
> 再强调一遍：他这个品牌的电脑的超级管理员账号 Administrator 是隐藏的，卖给你的时候那个账户没有完全的管理员权限

哥们你先别急没问你呢

**Reply by burningtnt** [reply]
你别慌，我只是友善的强调一遍（

**Comment by allMagicNB** [comment]
实在不行，~~重装解决99％问题~~

**Comment by radish-andy** [comment]
最近莫名其妙就好了

**Reply by SqrtTan** [reply]
6
